Makefile

```diff
@@ -102,6 +102,20 @@ vet:
 			echo "and fix them if necessary before submitting the code for reviewal."; \
 		fi
 
+# tools/godoctests/.bin/godoctests builds the custom analyzer to check for godocs for tests
+tools/godoctests/.bin/godoctests:
+	@cd tools/godoctests && $(GO_CMD) build -o .bin/godoctests .
+
+# vet-godoctests runs godoctests on the test functions. All output gets piped to revgrep
+# which will only return an error if a new function is missing a godoc
+vet-godoctests: bootstrap tools/godoctests/.bin/godoctests
```

Comment:
I tried checking out your branch and running this, but got:

```
ncc-PVXF6M0YT4:vault ncc$ make vet-godoctests
Installing/Updating golang.org/x/tools/cmd/goimports
Installing/Updating github.com/golangci/revgrep/cmd/revgrep
go generate -tags tools tools/tools.go
no required module provides package github.com/golangci/revgrep/cmd/revgrep; to add it:
        go get github.com/golangci/revgrep/cmd/revgrep
tools/tools.go:19: running "go": exit status 1
make: *** [bootstrap] Error 1
```

Reply:
thanks for catching that, I forgot to update go.mod

Reply:
Works now, ty!